content/en/docs/test.md

```diff
@@ -0,0 +1,201 @@
+    had one blank line, it would still show up as part of the previous list.
+2.  Numbered lists can have paragraphs or block elements within them.
+
+    Just indent the content to be even with the text of the bullet point, rather
+    than the bullet itself.
+
+    ```bash
+    $ ls -l
+    ```
+  
+  - And a sub-list after some block-level content
+
+
+### Checklists
+
+- [ ] This is a checklist item
+- [x] This is a selected checklist item
+
+## Code blocks
+
+You can create code blocks two different ways:
+
+- by indenting the entire code block 4 spaces from the content
+
+      this is a code block created by indentation
+      
+- by surrounding the code block with three back-tick characters on lines before
+  and after the code block.
+  
+  ```
+  this is a code block created by back-ticks
+  ```
+
+The back-tick method has some advantages.
+
+- It is more compact when viewing the source code.
+- It allows you to specify what language the code block is in, for syntax
+  highlighting.
+- It has a definite ending. Sometimes, the indentation method breaks with
+  languages where spacing is significant, like Python or YAML.
+
+To specify the language for the code block, put it directly after the first
+grouping of back-ticks:
+
+```bash
+$ ls -l
+```
+
+## Links
+
+To format a link, put the link text inside square brackets, followed by the
+link target in parentheses. [Link to Kubernetes.io](https://kubernetes.io/) or
+[Relative link to Kubernetes.io](/)
+
+You can also use HTML, but it is not preferred.
+<a href="https://kubernetes.io/">Link to Kubernetes.io</a>
+
+
+## Images
+
+To format an image, use similar syntax to [links](#links), but add a leading `!`
+character. The square brackets contain the image's alt text. Try to always use
+alt text so that people using screen readers can get some benefit from the
+image.
+
+![pencil icon](/static/images/pencil.png)
+
+An image can also be a link. This time the pencil icon links to the Kubernetes
+website. Outer square brackets enclose the entire image tag, and the link target
+is in the parentheses at the end.
+
+[![pencil icon](/static/images/pencil.png)](https://kubernetes.io)
+
+You can also use HTML for images, but it is not preferred.
+
+<img src="/static/images/pencil.png" alt="pencil icon" />
+
+
+## Tables
+
+Simple tables have one row per line, and columns are separated by `|`
+characters. The header is separated from the body by cells containing nothing
+but at least three `-` characters. For ease of maintenance, try to keep all the
+cell separators even, even if you heed to use extra space.
+
+| Heading cell 1 | Heading cell 2 |
+|----------------|----------------|
+| Body cell 1    | Body cell 2    |
+
+The header is optional. Any text separated by `|` will render as a table.
+
+Markdown tables have a hard time with block-level elements within cells, such as
+list items, code blocks, or multiple paragraphs. For complex or very wide
+tables, use HTML instead.
+
```

Comment:
This table doesn't seem to render correctly, but I can't see why: 
https://deploy-preview-8349--kubernetes-io-master-staging.netlify.com/docs/test/#tables